tests/misc_env/install_prereq.py

```diff
@@ -24,12 +23,11 @@
         "wget",
         "git",
         "python3-virtualenv",
-        "redhat-lsb",
         "python3-nose",
         "python3-pip",
     ],
 }
-deb_packages = ["wget", "git", "python-virtualenv", "lsb-release", "ntp"]
```

Comment:
We *do* need to keep this one for Ubuntu, because we use the `lsb_release` command.

Reply:
Noted. Will change it.